FiveCalls/FiveCallsTests/AppStateTests.swift

```diff
@@ -0,0 +1,76 @@
+import XCTest
+@testable import FiveCalls
+
+final class AppStateTests: XCTestCase {
+    
+    var sut: AppState!
```

Comment:
_nit:_ what is "sut"?

Reply:
**s**tate **u**sed for **t**est? 😂 I have no clue, I asked Claude to generate a test harness for the app state.

Reply:
Haha, well that's exciting!